src/server/lib/test/TestIntrospectionRequest.js

```diff
@@ -0,0 +1,56 @@
+const Test = require('./Test.js');
```

Comment:
Come per la PR feat/#11, se le libs non ci servono per i test in oggetto rimuoviamole dalla PR per ora.